pkg/apis/certmanager/v1/types_certificate.go

```diff
@@ -249,6 +253,38 @@ var (
 	RotationPolicyAlways PrivateKeyRotationPolicy = "Always"
 )
 
+// OutputFormatType specifies which additional key formats should be added to Kubernetes secrets.
+// Allowed values are `DER` or `CombinedPEM`.
+// When Type is set to `DER` an additional entry `key.der` will be created in the secret
+// containing the binary format of the key.
+// When Type is set to `CombinedPEM` an additional entry `tls-combined.pem` will be created in the secret
+// containing the PEM formatted certificate chain and key (tls.key + tls.crt concatenated)
+// +kubebuilder:validation:Enum=DER;CombinedPEM
+type OutputFormatType string
+
+const (
+	// AdditionalKeyOutputFormatDER requests that the DER binary format of the key
+	// is stored in the `key.der` key of a secret
+	AdditionalKeyOutputFormatDER OutputFormatType = "DER"
+
+	// AdditionalKeyOutputFormatDERKey is the name of the data entry in the Secret resource
+	// used to store the DER formatted private key
+	AdditionalKeyOutputFormatDERKey string = "key.der"
+
+	// AdditionalOutputFormatCombinedPEM requests that an entry containing both the PEM tls.crt and tls.key
+	// values concatenated is stored in the `tls-combined.pem` key of a secret
+	AdditionalOutputFormatCombinedPEM OutputFormatType = "CombinedPEM"
+
+	// AdditionalOutputFormatPEMKey is the name of the data entry in the Secret resource
+	// used to store the combined PEM certificate + key
+	AdditionalOutputFormatPEMKey string = "tls-combined.pem"
+)
+
+// AdditionalOutputFormat wraps an additional key output format type
+type AdditionalOutputFormat struct {
+	Type OutputFormatType `json:"type"`
```

Comment:
Add comment to the field here since it will be exported into docs for website/CRD.

Reply:
Let's follow up with another PR to fix this 😊